query-node/schema.graphql

```diff
@@ -65,55 +52,34 @@ type ChannelCategory @entity {
 
   channels: [Channel!] @derivedFrom(field: "category")
 
-  happenedIn: Block!
+  happenedIn: BigInt!
 }
 
-"Storage asset"
-union Asset = AssetUrl | AssetStorage
-
-"Asset stored at an external source"
-type AssetUrl @variant {
-  id: ID!
-
-  "The http url pointing to the media"
-  url: String!
-}
+"Asset availability representation"
+enum AssetAvailability {
+  "Asset is available in storage"
+  ACCEPTED,
 
-"Asset was never fully uploaded."
-type AssetNeverProvided @variant {
-  happenedIn: Block!
-}
+  "Asset is being uploaded to storage"
+  PENDING,
 
-"Asset was deleted and is no longer available."
-type AssetDeleted @variant {
-  happenedIn: Block!
-}
+  "Asset is referencing URL"
+  URL,
```

Comment:
I think we will be better off with removing `URL` as explicit state and use `ACCEPTED` for that? We will want to query entities with assets either accepted by storage or provided with URL and Hydra won't allow to express that (as there's no `OR` predicate) with this setup

Reply:
`URL` option has been removed.